src/app/home/event-user-list/event-user-list.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit, Input, ViewChild, Output, EventEmitter } from '@angular/core';
+import { ClrModal } from '@clr/angular';
+import { Progress } from 'src/app/data/progress';
+import { User } from 'src/app/data/user';
+
+interface userListUser extends User {
```

Comment:
```suggestion
interface UserListUser extends User {
```

Should be in PascalCase